static/src/js/components/BoundingRectangleField/BoundingRectangleField.jsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types'
 import Button from 'react-bootstrap/Button'
 
 import './BoundingRectangleField.scss'
+import CustomWidgetWrapper from '../CustomWidgetWrapper/CustomWidgetWrapper'
```

Comment:
The style import should be the last import in the file, `import './BoundingRectangleField.scss'`. We do this for readability and consistency across the repo